criteria/THIRD_PARTY_DEPENDENCIES.MD

```diff
@@ -0,0 +1,39 @@
+# Inclusion of third party dependencies complies with ASF 3rd Party License Policy
+
+Q&A and context for evaluators.
+
+## How should the evaluator handle a license that doesn’t appear at all on the ASF page?
+
+When FOLIO Module Evaluator tool encounters a dependency with an unknown license, it will report that the module is not compliant with FOLIO policy.  The evaluator should then fail the third-party license criterion.
+
+Technical Council [process](https://github.com/folio-org/tc-module-eval/blob/master/LICENSE_LIST_MANAGEMENT.md) will keep the evaluator tool’s lists of licenses up to date with the ASF source lists and FOLIO policy.
+
+## What should TC do when a new module includes a problem dependency that is already included in existing modules (that went through TCR, or pre-dated it)?
+
+This advice may be adjusted after CC (and TC) receives the requested legal advice on these issues.
+
+1. Raise the issue with the PO & dev lead to see if there is an alternative dependency (with a safe license) that would serve the same need.  If so, ask the team to switch.
+
+    1. Is that fair to ask at the evaluation stage where the module is already written?  Only if TC gives reasonable notice.  TC will communicate about this [at the start of each release cycle](https://folio-org.atlassian.net/wiki/spaces/TC/pages/227082271/Recurring+Calendar) (as we do in suggesting RFCs) to avoid a surprise.
+
+    1. Is that fair in this context of dependencies that are already part of the project?
+
+        1. Yes, because it prevents us from making the legacy problem worse while we figure out how to address it.  
+
+        1. And also because the alternative solution found for the new module may help solve the same problem in the old modules – or if not, at least demonstrate that the problem is a difficult one to solve with a safe license.
+
+1. If the PO & dev lead cannot find an alternative, they may present an argument to CC on why the inclusion of the dependency in the new module outweighs the license risks.  This may be a lengthy review process, during which TC's module evaluation would pause.
```

Comment:
I wonder if it is possible to ask dependency libraries to change their licensing and if not, could the dev team build their own solution.

Reply:
I agree both of these would be good solutions.  I'm not sure how fast either could get done, but I suppose it may be no slower than finding an alternative dependency.  Do you have any suggested language?

Reply:
Subgroup consensus is that it's true these two are possible "alternative"s alluded to above, but to spell them out in the doc would be somewhat unlikely to be successful.